MetaMorpheus/EngineLayer/SpectralMatch.cs

```diff
@@ -225,9 +235,13 @@ public void ResolveAllAmbiguities()
                 }
             }
 
-            // TODO: technically, different peptide options for this PSM can have different matched ions
-            // we can write a Resolve method for this if we want...
-            MatchedFragmentIons = BioPolymersWithSetModsToMatchingFragments.First().Value;
+            // Technically, different peptide options for this PSM can have different matched ions
+            // However, writing out all the matched ions for all the peptide options would break excel
+            // Instead, we set MatchedFragmentIons as the ions matched to the best peptide option
+            if (this is CrosslinkSpectralMatch)
+                MatchedFragmentIons = BioPolymersWithSetModsToMatchingFragments.Values.First();
```

Comment:
can we add a comment here that in classic search this kind of selection made for problematic results and should be evaluated carefully at som etimein the future.